db/fixtures/miq_shortcuts.yml

```diff
@@ -31,8 +31,8 @@
   :startup: true
 - :name: services
   :description: Services / My Services
-  :url: /service/explorer
-  :rbac_feature_name: service
+  :url: /service/show_list
+  :rbac_feature_name: service_show_list
```

Comment:
@jeffibm Not sure if this requires a data migration or not - in the past when harpreet did de-explorizations, shortcuts were removed, which required a data migration, but this is just a change, which might be handled fine by seeding.  If you look in the schema repo and her old PRs you can see what she did previously.

Reply:
Right those were only for the product features - this is shortcuts.

Reply:
Example PR where harpreet had to change it for the actions de-explorization: https://github.com/ManageIQ/manageiq-schema/pull/547/files

Reply:
@jeffibm Did you already handle the bookmark changes?

I know you and I worked on the migration for the privileges, but don't think we handled this one.